### PR TITLE
Resolve Symfony 3.4 deprecation warnings

### PR DESCRIPTION
**Pull request by mikelerch**
I suggest merging this into a new `2.0` branch to ensure older applications don't break due to php / package versions being raised.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,16 +16,18 @@
...
-        "symfony/yaml": ">=2.0",
-        "kriswallsmith/assetic": ">= 1.0",
-        "symfony/framework-bundle": ">=2.0.9"
+        "php": "^7.1",
+        "symfony/finder": "^3.4",
+        "symfony/yaml": "^3.4",
+        "kriswallsmith/assetic": "^1.4",
+        "symfony/translation": "^3.4",
+        "symfony/framework-bundle": "^3.4",
+        "symfony/templating": "^3.4"
```
Comment (mikelerch) [inline composer.json]: `symfony/templating` and `symfony/translation` are no longer implicitly required by `symfony/framework-bundle`.

**Comment by wesselvanderlinden** [comment]
nice +1

File: composer.json (hunk 1 above)
Comment (mikelerch) [reply composer.json]: Should probably lower the PHP version again to keep support for 5.x, so no upgrade is forced.

File: composer.json [hunk 2]
```diff
@@ -20,12 +20,14 @@
         "symfony/finder": ">=2.0",
         "symfony/yaml": ">=2.0",
         "kriswallsmith/assetic": ">= 1.0",
-        "symfony/framework-bundle": ">=2.0.9"
+        "symfony/translation": ">=2.0.9",
+        "symfony/framework-bundle": ">=2.0.9",
+        "symfony/templating": ">=2.0.9"
```
Comment (mikelerch) [inline composer.json:25]: `symfony/templating` and `symfony/translation` are no longer implicitly required by `symfony/framework-bundle` in newer versions, which is why they are now explicit dependencies.

**Pull request merged**